lambdas/data-migration2/src/granulesAndFiles.ts

```diff
@@ -140,20 +143,30 @@ export const migrateFileRecord = async (
 
 /**
  * Migrate granule and files from DynamoDB to RDS
- * @param {AWS.DynamoDB.DocumentClient.AttributeMap} dynamoRecord
- * @param {GranulesAndFilesMigrationResult} granuleAndFileMigrationResult
- * @param {Knex} knex
- * @param {number} loggingInterval
+ * @param {Object} params
+ * @param {AWS.DynamoDB.DocumentClient.AttributeMap} params.dynamoRecord
+ * @param {GranulesAndFilesMigrationSummary} params.granuleAndFileMigrationSummary
+ * @param {Knex} params.knex
+ * @param {number} params.loggingInterval
+ * @param {any} params.stream
  * @returns {Promise<MigrationSummary>} - Migration summary for granules and files
  */
-export const migrateGranuleAndFilesViaTransaction = async (
+export const migrateGranuleAndFilesViaTransaction = async (params: {
   dynamoRecord: AWS.DynamoDB.DocumentClient.AttributeMap,
   granuleAndFileMigrationResult: GranulesAndFilesMigrationResult,
   knex: Knex,
-  loggingInterval: number
-): Promise<GranulesAndFilesMigrationResult> => {
-  const files = dynamoRecord.files ?? [];
+  loggingInterval: number,
+  stream: any,
```

Comment:
I think we can give this a type of `Writable` from the `stream` package. Also, we should give this variable a more descriptive name, like `errorsLogWriteStream`